.travis.yml

```diff
@@ -1,7 +1,17 @@
 language: go
 go:
-  - 1.9.x
   - 1.10.x
 
+sudo: required
+
+env:
+- CHANGE_MINIKUBE_NONE_USER=true
+
 before_script:
-  - go get github.com/hashicorp/consul && go install github.com/hashicorp/consul
\ No newline at end of file
+- go get github.com/hashicorp/consul && go install github.com/hashicorp/consul
```

Comment:
This will install the latest version of consul. We should control its version and download a zip.

Reply:
I would rather make a separate pull request that will remove usage of testutil Consul package because it is essentially fucked up (depends on consul binary in $PATH). We can mock the Consul client and do the integration testing only on k8s which, by the way, are more complete.

Reply:
So what we will test here? If we mock consul then it will not be an integration test.

Reply:
Running Consul may be a pain, but I agree with Janisz here.

Reply:
By integration testing on k8s I mean testing on pod with Consul and hook added do lifecycle events. It will also test communication with k8s API.